src/client/datascience/ipywidgets/ipyWidgetScriptSource.ts

```diff
@@ -143,7 +133,7 @@ export class IPyWidgetScriptSource implements ILocalResourceUriConverter {
         if (!this.uriConversionPromises.has(key)) {
             this.uriConversionPromises.set(key, createDeferred<Uri>());
             // Send a request for the translation.
-            this.postInternalMessageEmitter.fire({
```

Comment:
I may be remembering wrong but I believe the internal event didn't go to the webview (just the listeners)? I think this just means the webview will get a message it doesn't care about.

Reply:
Based on the code, it didn't make any difference, both were identical, hence removed it.
I.e. absolutely 0 change